sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/MapStateImplWithTTL.scala

```diff
@@ -231,7 +232,8 @@ class MapStateImplWithTTL[K, V](
     // ttlExpiration
     Option(retRow).flatMap { row =>
       val ttlExpiration = stateTypesEncoder.decodeTtlExpirationMs(row)
-      ttlExpiration.map(expiration => (stateTypesEncoder.decodeValue(row), expiration))
+      ttlExpiration.map(expiration => (
```

Comment:
nit: shall we move ( down? Also `.map { expiration =>` so that it's not confused with ( and ) for tuple.

Reply:
Done